app/models/service_template_transformation_plan_task.rb

```diff
@@ -132,20 +132,21 @@ def transformation_log
 
   # Intend to be called by UI to display transformation log. The log is stored in MiqTask#task_results
   # Since the task_results may contain a large block of data, it is desired to remove the task upon receiving the data
-  def transformation_log_queue(userid = nil)
+  def transformation_log_queue(userid = nil, log_type = 'v2v')
+    raise "Transformation log type '#{log_type}' not supported" unless %w(v2v wrapper).include?(log_type)
```

Comment:
Minor nitpick, I'd vote for a `def valid_transformation_log_types` method that returned this array instead of hardcoding it. That way down the road, we could just modify that method and/or overload it if needed. Not a showstopper, though.